src/Stripe.net/Entities/AccountLinks/AccountLink.cs

```diff
@@ -1,7 +1,6 @@
 namespace Stripe
 {
     using System;
-    using System.Collections.Generic;
```

Comment:
@ob-stripe Just wanted to make sure that this can't be a breaking change right?

Reply:
No, if it compiles then it's fine. I wish there was a StyleCop rule to force us to remove unnecessary imports.